src/main/java/ee/tuleva/onboarding/comparisons/returns/provider/MarketReturnProvider.java

```diff
@@ -0,0 +1,43 @@
+package ee.tuleva.onboarding.comparisons.returns.provider;
```

Comment:
Could we remove the duplication here by having, say, an `IndexReturnProvider`?

Reply:
Need to think about it a bit. Would prefer composition over inheritance here.